compiler/src/dmd/expressionsem.d

```diff
@@ -6892,8 +6930,6 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
         AggregateDeclaration ad = f.isMemberLocal();
         if (f.needThis())
             e.e1 = getRightThis(e.loc, sc, ad, e.e1, f);
-        if (e.e1.op == EXP.error)
-            return setError();
```

Comment:
Why is this removed?

Reply:
Because it was added to fix a symptom of the bug this PR is fixing. Take a look at: https://github.com/dlang/dmd/pull/10123